config.json

```diff
@@ -444,6 +444,19 @@
       "uuid": "55b87cc2-c3a0-438a-b2b6-116ad6356802"
     },
     {
+      "core": true,
+      "difficulty": 4,
+      "slug": "sublist",
+      "topics": [
+        "pointers",
+        "arrays",
+        "searching",
+        "control_flow_if_statements",
+        "control_flow_loops"
+      ],
+      "uuid": "928e9860-0df9-5180-8529-7024c991b3616adc21a"
```

Comment:
does this need an "unlocked_by" field?

Reply:
Let's go with non-core. They want to have very few core exercises (I think 6-8?) so better to not burn them without a good reason.

Reply:
Ok, `palindrome-products` seems like a reasonable exercise to unlock this one, in my opinion

Reply:
Yes, `palindrome-products` seems like a good choice to unlock this IMO